system-int-test/test/cmr/system_int_test/search/service/collection_service_search_test.clj

```diff
@@ -269,4 +281,15 @@
       (au/associate-by-concept-ids token serv9-concept-id [{:concept-id (:concept-id coll4)}])
       (index/wait-until-indexed)
       ;; after service association is made, has-transforms is true
-      (assert-collection-atom-json-result coll4 {:has-transforms true} [serv9-concept-id]))))
+      (assert-collection-atom-json-result coll4 {:has-transforms true} [serv9-concept-id]))
+
+    (testing "SubsetType with no 'Spatial' results in spatial-subsetting being false"
```

Comment:
Move this check up to line 270 above, i.e. associate service 10 with coll3 and check `has-spatial-subsetting` is false; then associated service8 with coll3, verify that `has-spatial-subsetting` would be true after that.